public/app.js

```diff
@@ -19,15 +19,31 @@ class Controller {
         document.querySelector(DOMstrings.graphContainer).innerHTML = "";
     }
 
+    static displayCodingProgress() {
+        // Add the coding progress section to the UI
+        UIController.addCodingProgressSection();
+        // Get data for coding progress table
+        let unsubscribeFunc = DataController.watchCodingProgress(UIController.updateProgressUI);
+        DataController.registerSnapshotListener(unsubscribeFunc);
+    }
+
+    static displaySelectedProject(project) {
```

Comment:
```suggestion
    static displayProject(project) {
```
On reading the name displayProject, my expectation is that you'll be passing in the project name to be displayed.

On reading the name displaySelectedProject, my expectation is that the function will somehow already know which project is selected, presumably by a class or global variable somewhere, so this function will take 0 arguments.

What you've implemented is the right thing to do, so I think better to call it `displayProject`